contents/blog/open-source-business-models.md

```diff
@@ -0,0 +1,68 @@
+
+## Option #3: Open core
+This option requires you to build and maintain two versions of your product: the open-source version (under an MIT or Apache-type license), and a ‘source-available’ version with features designed for larger businesses. Your customers can deploy the source-available version on their own infrastructure, but they need to pay you to use those added features. This is the approach that PostHog uses; you can access our [community version](http://github.com/PostHog/) on GitHub.
+
+The major advantage of this approach — and one of our biggest selling points — is that you don’t need to process user data through 3rd parties. This makes it easier to gain adoption in larger companies that hold data privacy as a core part of their ethos.
+
+<iframe src="https://giphy.com/embed/3HELB2Qwfu9dV1ZGYY" width="480" height="282" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/3HELB2Qwfu9dV1ZGYY">via GIPHY</a></p>
+
+
+The downside is that you have to build and maintain two versions of your product: the community version and the source-available version. This means hiring a larger, costlier engineering team and possibly raising venture capital at the start to fund your operations. Due to these factors, it might take longer for you to get to revenue. And because you’re maintaining two versions of your product, you’ll need to clearly indicate which features contributors can work on and which ones are off-limits.
+
+At PostHog, we launched a (paid) hosted version first before moving to an open-core model and allowing self-hosting. Many other companies are experimenting with the concept of ‘buyer-based open core,’ which means that you [charge differently](https://www.heavybit.com/library/video/commercial-open-source-business-strategies/) for features depending on who cares the most about them. Company executives always want more functionality and tend to be less price-sensitive than, say, a developer or individual contributor, so there’s more room to expand your codebase and make money. The downside is that it takes a lot of work to separate your open-source code from your proprietary code.
+
+<iframe src="https://giphy.com/embed/9Y74y9h95QBizsNfd8" width="480" height="480" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/paidoff-trutv-paid-off-michael-torpey-9Y74y9h95QBizsNfd8">via GIPHY</a></p>
+
+## Option #4: Donations
+This approach is great if you’re merely maintaining your product as a hobby and are fine with earning ice cream money each month. It quickly becomes unfeasible if you want to maintain an entire engineering team on donations. You can enjoy quite a bit of success going the donation route if you have a stellar product, but these are outliers and shouldn’t be used as revenue benchmarks. Besides, if you already have a great product that’s raking in donations, you probably shouldn’t be reading this post.
```

Comment:
The tone of this is pretty good.

The only thing, with our audience in mind, that stands out in a couple of places - don't downplay the importance of other reasons that people might build open source- some of the world's best software makes no money and literally is used by hundreds of millions of end users.  "merely maintaining" would grate me as a technical person - it's a lot of work, and helps many people out

Reply:
Thanks - tweaked the wording in the file.